identity/prebid-identity.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: page_v2
+title: Prebid User Identity
+description: What is Prebid User Identity
+sidebarType: 9
+---
+
+# Prebid User Identity Overview
+
+`DRAFT`
+
+Prebid's aim is to enable the protection of user privacy while still supporting publisher's ability to make revenue,
+keeping the Open Web healthy.
+
+To do this, Prebid offers a number of identity-related products that encourage awareness of privacy regulations such as GDPR, CCPA, and COPPA. The most important projects are:
+
+- [Prebid.js User Identity Module](/dev-docs/modules/userId.html) - supporting more than 20 different flavors of global IDs with different features, publishers can work with.
+- [SharedId and PubCommon ID](/identity/sharedIdPubCommon.html) - Prebid's native hosted ID offerings are simple, free, robust, and privacy-minded.
+- Coming soon: UID2
+
+## Prebid.js and Identity
+
+Publishers have several ways to include user identity as part of
+the header bidding auction:
+
+1. Install one or more [User ID modules](/dev-docs/modules/userId.html) - these modules obtain
+the user's ID from the service and make it available to participating bidders. Publishers
+can place [permissons](/dev-docs/modules/userId.html#permissions) on which bidders receive which IDs.
+2. Install the [ID Import Module](/dev-docs/modules/idLibrary.html) - this can be
+used to generates a map of identities present on the page.
+3. Pass registered [User First Party Data](/features/firstPartyData.html) to bidders such as interests to bidders for more relevant advertising.
```

Comment:
strike the second "to bidders" so it reads: "pass registered user first party data to bidders such as interests for more relevant advertising."